engineering-principles.md

```diff
@@ -0,0 +1,19 @@
+# Engineering Principles
+
+We like to apply the following engineering principles in the way we develop our services and digital products.
+
+* Reusable services as engineering products
+* API-first
+* Cloud-native
+* Real-time data and monitoring
+* [Accessibility](frontend/accessibility.md) at the heart of everything
+* Going beyond the screen
+
+We approach those principles in the following manner:
+
+* Development of services over maintaining infrastructure and manual scaling. Functions over microservices, microservices over monoliths, managed services over self-managed services. [12-factor apps](https://12factor.net/) where applicable
+* Use of and contribute to open source software. [Coding in the open as our default setting](open-source.md).
+* Continuous delivery backed up by automated testing. Deploy to production systems multiple times a day.
+* Real-time monitoring of live products through dashboards (e.g. Grafana), logs (e.g. Loggly, CloudWatch) and alerts (e.g. Wormly, Slack)
```

Comment:
Slack's just a channel for monitors to ping to?

Reply:
True, have taken off Slack, there's no need to list it here.